package.json

```diff
@@ -13,7 +13,7 @@
     "start:zombienet-liquidity-assets-script": "yarn build:scripts && node ./scripts/build/testNetworkLiquidAssets.js",
     "lint": "substrate-dev-run-lint",
     "lint:fix": "substrate-dev-run-lint --fix",
-    "test": "NODE_ENV=test substrate-exec-jest --detectOpenHandles",
+    "test": "NODE_ENV=test substrate-exec-jest --silent --detectOpenHandles",
```

Comment:
Was this pushed up accidently?

Reply:
good catch. Was using during debugging/testing